engines/bastion_katello/app/assets/javascripts/bastion_katello/organizations/organization.factory.js

```diff
@@ -14,7 +14,20 @@ angular.module('Bastion.organizations').factory('Organization',
         return BastionResource('/katello/api/v2/organizations/:id/:action',
             {id: '@id'},
             {
-                update: { method: 'PUT'},
+                get: {
+                    method: 'GET',
+                    transformResponse: function (data) {
+                        // Some fields cannot be PUT back
+                        var org;
+
+                        org = angular.fromJson(data);
+                        delete org.users;
+                        delete org.parent_id;
+                        delete org.parent_name;
+                        return org;
```

Comment:
right, walden hit this very similarly with hosts recently and I liked the approach he ended up taking much better.  Editing the object in the GET seems really hacky (or even more hacky) and means if we ever actually need to use  users, parent_id or parent_name, we can't.  

https://github.com/Katello/katello/blob/master/engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/content-host-details.controller.js#L60-L72

I don't mean to be a stickler, but this just seems like the wrong approach.